acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -6,32 +6,5 @@ def ruby_command(host)
       end
       module_function :ruby_command
```

Comment:
we have ruby_command defined in beaker-puppet. We might be able to use the method defined there rather than this one

https://github.com/puppetlabs/beaker-puppet/blob/master/lib/beaker-puppet/install_utils/foss_utils.rb#L1406

Reply:
Great, I was wondering about that.

Reply:
Done.